controllers/idler/idler_controller.go

```diff
@@ -145,7 +145,7 @@ func (r *Reconciler) ensureIdling(logger logr.Logger, idler *toolchainv1alpha1.I
 					podLogger.Info("Pod deleted")
 				}
 				// By now either a pod has been deleted or scaled to zero by controller, idler Triggered notification should be sent
-				if err := r.createNotification(logger, idler); err != nil {
+				if err := r.createNotification(logger, idler, pod.Name); err != nil {
```

Comment:
instead of passing the actual pod name to the `createNotification` func, I believe it would be better if we can pass the controller name (ie, the name parent Deployment, StatefulSet, etc.), as the notification sent to the user will be more explicit. See the call to `scaleControllerToZero` on line 136.

For example: 

> In accordance with the usage terms of Developer Sandbox, we have reduced the number of instances of your
        application my-app to zero (0)

and not something like

> In accordance with the usage terms of Developer Sandbox, we have reduced the number of instances of your
        application my-app-1a2b3c to zero (0)

Reply:
yeah, but that's applicable only in those cases when the pod is managed by a controller (eg. Deployment), but there could be cases when there is a single pod (which is not managed by any controller) that just got deleted - in such situation we can provide the pod name only, nothing else. 
In other words:
* if the pod is managed by a controller, let's provide the name of the controller
* if the pod is not managed by a controller, then let's provide the name of the pod

Reply:
yes, I forgot to mention that if the pod has no controller, then we should pass it as-is.

Reply:
Okay , i understand,  will it be the same case for daemonset, job controller type also  ?

Reply:
There are a few options.
1. We just call it an application as we do now. We just add a name of the controller or standalone pod:
```
In accordance with the usage terms of Developer Sandbox, we have reduced the number of instances of your
application {{.ApplicationName}} to zero (0)
```
2. We introduce two params. Application Type and Application Name. Something like:
```
In accordance with the usage terms of Developer Sandbox, we have reduced the number of instances of your
{{.ApplicationType}} {{.ApplicationName}} to zero (0)
```
Where `{{.ApplicationType}}` is `Deployment`, `ReplicaSet` or `Pod` (if no controllers). And `{{.ApplicationName}}` is the deployment, replicaset, pod, etc name.

I think the second option would be even better.

BTW, please update the template itself so the controller/pod name is used here: https://github.com/codeready-toolchain/host-operator/blob/master/deploy/templates/notificationtemplates/sandbox/idlertriggered/notification.html#L50

Reply:
Just noticed the other PR: https://github.com/codeready-toolchain/host-operator/pull/878
Please add it to the description of this PR so it's harder to miss :)

Reply:
Added the related PR here also 
 
Okay agree for second option, making the changes to the PR

Reply:
Updated the PR